packages/atomic/src/components/atomic-result-list/atomic-result-list.tsx

```diff
@@ -121,18 +122,123 @@ export class AtomicResultList implements InitializableComponent {
       });
   }
 
-  private get results() {
-    return this.resultListState.results.map((result) => (
+  private getTemplate(result: Result) {
+    return this.resultTemplatesManager.selectTemplate(result) || '';
+  }
+
+  private buildResult(result: Result) {
+    return (
       <atomic-result
         key={`${result.raw.permanentid}${this.resultListState.searchUid}`}
         result={result}
         engine={this.bindings.engine}
         display={this.display}
         density={this.density}
         image={this.image}
-        content={this.resultTemplatesManager.selectTemplate(result) || ''}
+        content={this.getTemplate(result)}
       ></atomic-result>
-    ));
+    );
+  }
+
+  private buildListResults() {
+    return this.resultListState.results.map((result) =>
+      this.buildResult(result)
+    );
+  }
+
+  private buildTableResults() {
+    const fieldColumns = Array.from(
+      parseHTML(
+        this.getTemplate(this.resultListState.results[0])
+      ).querySelectorAll('atomic-table-element')
+    );
+
+    return (
+      <table>
+        <thead>
+          <tr>
+            <th>Description</th>
+            {fieldColumns.map((column) => (
+              <th>{column.getAttribute('label')}</th>
+            ))}
+          </tr>
+        </thead>
+        <tbody>
+          {this.resultListState.results.map((result) => (
+            <tr>
+              <td>{this.buildResult(result)}</td>
+              {fieldColumns.map((column) => (
+                <td>
+                  <atomic-table-cell
+                    result={result}
+                    content={column.innerHTML}
```

Comment:
Does this mean we can put absolutely whatever we want in the table cell as an element ?

AKA: Would this work:

```
<atomic-table-element label="Author">
  <span>Hello world</span>
  <my-custom-component-that-display-weird-stuff />
</atomic-table-element>
```

If so 👍.

Reply:
Yes, that's what it means.